chainerrl/agents/ppo.py

```diff
@@ -0,0 +1,265 @@
+import chainer
+from chainer import cuda
+import chainer.functions as F
+import copy
+
+
+from chainerrl import agent
+from chainerrl.misc.batch_states import batch_states
+
+
+def _F_clip(x, x_min, x_max):
```

Comment:
I see why this function is needed, but could you give it a better name, say, `elementwise_clip`? `_F_clip` is confusing with `F.clip` and doesn't describe what it does.